src/components/common/fullCalendar/DayHeaderContent.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from '@emotion/styled';
+import { DayHeaderContentArg } from '@fullcalendar/core';
+
+interface DayHeaderContentProps {
+	arg: DayHeaderContentArg;
+	currentView: string;
+	today: string;
+}
+
+function DayHeaderContent({ arg, currentView, today }: DayHeaderContentProps) {
+	const isTimeGridDay = currentView === 'timeGridDay';
+	const day = new Intl.DateTimeFormat('en-US', { weekday: isTimeGridDay ? 'long' : 'short' }).format(arg.date);
+	const date = arg.date.getDate();
+	const isToday = arg.date.toDateString() === today;
+
+	return (
+		<div>
+			{!isTimeGridDay ? (
+				<>
+					<WeekDay isToday={isToday}>{day}</WeekDay>
+					{currentView !== 'dayGridMonth' && <WeekDate isToday={isToday}>{date}</WeekDate>}
+				</>
+			) : (
+				<DayLayout>
+					<WeekDate isToday={false}>{date}일</WeekDate> <WeekDay isToday={false}>{day}</WeekDay>
+				</DayLayout>
+			)}
+		</div>
+	);
+}
+
+const DayLayout = styled.div`
+	display: flex;
+	gap: 1.2rem;
+	align-items: flex-end;
+	margin-left: 0.8rem;
+`;
+
+const WeekDay = styled.div<{ isToday: boolean }>`
+	${({ theme }) => theme.fontTheme.CAPTION_02};
+	color: ${(props) => (props.isToday ? props.theme.palette.BLUE_DISABLED : props.theme.palette.GREY_04)};
```

Comment:
```suggestion
	color: ${({ isToday, theme }) => (isToday ? theme.palette.BLUE_DISABLED : theme.palette.GREY_04)};
```

여기 일케 더 간결하게 쓸 수 있을 것 같습니다!

Reply:
해당 부분 수정하였습니다 감사합니다!